modules/account-lib/src/coin/eth/iface.ts

```diff
@@ -3,15 +3,20 @@ import { KeyPair } from './keyPair';
 
 export interface Fee extends BaseFee {
   gasLimit: string;
+  eip1559?: EIP1559Fee;
```

Comment:
do we want gasPrice to also be an optionalParam here? not sure why eip1559 needs to be here if gasPrice wasnt before

Reply:
nvm

Reply:
can add